APP-MANAGER

```diff
@@ -545,10 +545,11 @@ _check_version_for_auto_updatable_apps() {
 		if [ -f "$argpath"/updater ]; then
 			_check_version_if_any_version_reference_is_somewhere
 			if [ "$AMCLI" = am ] && [ -d "$CACHEDIR"/am ] && grep -q " ◆ $arg	|" "$CACHEDIR"/am/version-args; then
-				OLDAPPVERSION=$(grep " ◆ $arg	|" "$CACHEDIR"/am/version-args | awk '{print $NF}')
+				OLDAPPVERSION=$(grep " ◆ $arg	|" "$CACHEDIR"/am/version-args | awk '{print $NF}' | tail -1)
```

Comment:
`awk 'END {print $NF}'` and that way you avoid an extra pipe and` tail -1`

Reply:
I added `tail -1` because I had the issue on GhostBSD

Reply:
thank you https://github.com/ivan-hc/AM/pull/1209/commits/3f0da5a1ee8286fb97c802ec39301e893e681765